tasks/db_deploy/README.md

```diff
@@ -62,13 +62,13 @@ standards expected.
    migration from version x to version y. The `migrate_sql.py` file should have
    the SQL needed to perform the migration. See previous versions for examples
    on how the SQL should be formatted and expressed in functions. The unit test
-   files `test_migrate_vY.py` and `test_migrate_sql_vY.py` should contain unit
+   files `test_migrate_vy.py` and `test_migrate_sql_vy.py` should contain unit
```

Comment:
Wondering if you had a find replace mishap since the rest of the paragraph has an uppercase `Y` instead of lowercase `y` for the names.

Reply:
It switches a lot... See line 62. I think capitals are clearer, so I'll do a switch on everything.